### ISSUE TITLE
I can’t boot into the installation for macOS Big Sur on my laptop

### ISSUE THREAD

**Issue by beginnerhacki** [rlindsberg/Hackintosh-On-MSI-GL62m-7REX#26]
This thing comes up and I’m a beginner on this topic. So I have a Dell Inspiron 5000 series 15.6 with an intel core i7 nvidia GeForce 920M and 8 gigs of ram. My clover loads perfectly but then when I choose to boot to the macos it just gets stuck there: 
![image](https://user-images.githubusercontent.com/78041986/105884951-2d8f2b00-6011-11eb-8a80-92abc1624418.jpg)
![image](https://user-images.githubusercontent.com/78041986/105885013-3f70ce00-6011-11eb-830e-4afbe276fdfd.jpg)
Can anybody help me?

**Comment by jbwharris** [comment]
Start over [here with OpenCore](https://dortania.github.io/OpenCore-Install-Guide/). Using Clover isn't the way forward. OpenCore is really well documented and will help you through the process if you follow the guide closely.

**Comment by beginnerhacki** [comment]
I tried with open core too but it didn’t work. I don’t know why it might have been because in wanted to dual boot both windows and Mac so I made two partitions the one for windows and the other for Mac. Well I tried the open core one but there was still the clover one and I didn’t knew what to do so I erased one partition which I didn’t know what was and then my whole windows partition and stuff disappeared and there was only the macos on the usb stick. So then eventually I gave up and reinstalled windows 10 and made all the partitions again back to one
